tests/main/snap-multi-service-failing/task.yaml

```diff
@@ -0,0 +1,10 @@
+summary: >
```

Comment:
What does `>` do? If it is something non-essential I'd say it's better to be obvious rather than fancy here